package.json

```diff
@@ -10,6 +10,8 @@
     "lodash": "^4.17.21",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-material-ui-form-validator": "^2.1.4",
+    "react-native-material-textfield": "^0.16.1",
```

Comment:
Why have you install the package for just a form filed .. Is there was any issue customizing the Material UI text filed ..?